src/Compute/Compute/ChangeLog.md

```diff
@@ -23,6 +23,8 @@
 * Added update functionality in `Update-AzVmss` for parameters `SecurityType`, `EnableSecureBoot`, and `EnableVtpm` for the parameter set with the Put operation.
 * Upgraded Azure.Core to 1.35.0.
 * [Breaking change] Removed unversioned and outdated linux image aliases of `CentOS`, `RHEL`, `UbuntuLTS` and `Debian`.
+* [Breaking change] `New-AzVmss` will default to `OrchestrationMode` set as  `Flexible` if it is not set as `Uniform` explicitly.
```

Comment:
Can you add or update the existing vmss flex examples in help docs (NEw-AzVMss.md I think has one) to show this new functionality and how users can use it?

Reply:
new example added